package.json

```diff
@@ -121,7 +121,7 @@
     "tiny-emitter": "^2.1.0",
     "tough-cookie": "^4.0.0",
     "underscore.string": "^3.3.4",
-    "uploadfs": "^1.24.3",
+    "uploadfs": "github:apostrophecms/uploadfs#pro-8109-upgrade-dependencies",
```

Comment:
TODO: use version 6 when https://github.com/apostrophecms/uploadfs/pull/103 is merged

Reply:
exactly, sorry,

TODO: use version 1.25.1*